Automattic/vip-go-mu-plugins
# CANTINA-832: Search: Remove healthcheck for counts

## Description
Remove the healthcheck job for counts since it's not being utilized. We already have the validate-contents job which is self-healing.

## Changelog Description
### Plugin Updated: Enterprise Search

Remove healthcheck job for counts.

## Checklist

## Steps to Test
1) Check out PR
2) Enable healthchecks: `add_filter( 'enable_vip_search_healthchecks', '__return_true' );`
3) `wp cron event list` Check to see that `vip_search_healthcheck` doesn't exist in the list
